.github/workflows/ci.yml

```diff
@@ -4,24 +4,21 @@ on:
   pull_request:
     branches:
       - main
+  push:
+    branches:
+      - main
 
 jobs:
-  build-test:
+  build:
+    name: Build and test
     runs-on: ubuntu-latest
 
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
-
-      - name: Setup NodeJS 14
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
```

Comment:
Turns out we don’t need to specify a name for this, since the linked action will have a name already. Naming them is to customize how we see them, but I assume we are not there yet.